### ISSUE TITLE
GitHub Logo Not Visible in Footer Dark Mode

### ISSUE THREAD

**Issue by lovelymahor** [Nactore-Org/Nacto-Care-App#307]
### Describe the feature

The GitHub logo in the footer is not visible when the website is in dark mode. This affects the visibility and accessibility of our social media links, particularly for users who prefer dark mode.

### Add ScreenShots

![Screenshot (278)](https://github.com/user-attachments/assets/9fc10e68-1e4b-4e08-9ec0-78ad53352d3a)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by lovelymahor** [comment]
Please assign it to me I want to work on this issue. Also, please add the label GSSOC and the level that you find appropriate for this issue. thank you!
